### PR TITLE
Replace Go Doc URL with `pkg.go.dev`

### PR DESCRIPTION
**Pull request by jihoon-seo**
This PR
- supercedes https://github.com/kubernetes/website/pull/31319#issuecomment-1013882212
- is relevant with #31318

CC @tengqm (from https://github.com/kubernetes/website/pull/31319#issuecomment-1013882212)

### CODE REVIEW

**Review by divya-mohan0209 — APPROVED** [review]
approve

File: content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-certs.md [hunk 1]
```diff
@@ -173,7 +173,7 @@ The built-in signer is part of [`kube-controller-manager`](/docs/reference/comma
 
 To activate the built-in signer, you must pass the `--cluster-signing-cert-file` and `--cluster-signing-key-file` flags.
 
-If you're creating a new cluster, you can use a kubeadm [configuration file](https://godoc.org/k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm/v1beta3):
+If you're creating a new cluster, you can use a kubeadm [configuration file](https://pkg.go.dev/k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm/v1beta3):
```
Comment (bradtopol) [inline content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-certs.md:176]: Great job! This removes an unnecessary redirect.

**Comment by bradtopol** [comment]
Great job! The redirects are removed and all the new links work properly

**Pull request merged**